### ISSUE TITLE
Pivot Grid: wrong concatenated display folders

### ISSUE THREAD

**Issue by zenzeinet** [TabularEditor/TabularEditor3#777]
Pivot Grid: When measures are in multiple display folders (concatenated  with ";", shows the concatenated field instead of the correct split display folders

**Comment by otykier** [comment]
Yeah, this is a limitation of the Pivot Grid component we use. Will see if we can do something about it, but there are no guarantees (as it’s from a 3rd party library). You will see the same issue in some OLAP client tools.
